packages/react/docs/generated/generated_static_pages.json

```
@@ -69,60 +33,64 @@
+        "sectionContent": "\n      To use Hydrogen React, you need to authenticate with and make requests to the [Storefront API](/api/storefront-api). Refer to [Get started with Hydrogen React](/custom-storefronts/hydrogen-react#get-started-with-hydrogen-react) for instructions on how to get an access token and set up the Storefront API client.\n       ",
+        "sectionCard": [
           {
-            "title": "Utilities",
-            "sectionContent": "Utilities are reusable functions for common data manipulations performed on Storefront API data.",
-            "sectionCard": [
-              {
-                "subtitle": "Navigate to",
-                "name": "All Hydrogen React utility functions",
-                "url": "hydrogen-react/utilities",
-                "type": "library"
-              }
-            ]
+            "subtitle": "Navigate to",
+            "name": "Enable Storefront API Access",
+            "url": "/custom-storefronts/hydrogen-react#step-2-enable-storefront-api-access",
+            "type": "key"
           }
-        ],
+        ]
+      },
+      {
+        "type": "Generic",
+        "anchorLink": "versioning",
+        "title": "Versioning",
+        "sectionContent": "Hydrogen React is tied to specific versions of the [Storefront API](/api/storefront). For example, if you're using Storefront API version `2023-01`, then Hydrogen React versions `2023.1.x` are fully compatible. \n\n >Caution: \n>If the Storefront API version update includes breaking changes, then Hydrogen React includes breaking changes. Because the API version is updated every three months, breaking changes to Hydrogen React could occur every three months. \n\n Learn more about [API versioning](/api/usage/versioning)."
+      },
+      {
+        "type": "Generic",
+        "anchorLink": "components",
+        "title": "Components",
+        "sectionContent": "A component encapsulates all of the business logic and data parsing/processing for the concept it represents and outputs limited, sensible markup. Components provide defaults, but allow customizations and provide no visual styles, other than those provided natively by the browser.",
         "codeblock": {
-          "title": "Example",
+          "title": "Component example",
           "tabs": [
             {
               "title": "Component",
               "code": "import {ShopPayButton} from '@shopify/hydrogen-react';\n\nexport function MyProduct({variantId}) {\n  return <ShopPayButton variantIds={[variantId]} />;\n}\n",
               "language": "javascript"
-            },
+            }
+          ]
+        }
+      },
+      {
+        "type": "Generic",
+        "anchorLink": "hooks",
+        "title": "Hooks",
+        "sectionContent": "Hooks are functions that provide reusable, business and/or stateful logic. These functions allow you to leverage the business logic and state management functions used in the components with more flexibility.",
```

```suggestion
        "sectionContent": "Hooks are functions that provide reusable, business, and/or stateful logic. These functions allow you to leverage the business logic and state management functions used in the components with more flexibility.",
```

Thoughts on the oxford comma? haha

---

Always Oxford 🎩